test/conformance/memory/memory_adapter_hip.match

```diff
@@ -1,2 +1,20 @@
-urMemBufferCreateWithNativeHandleTest.Success/AMD_HIP_BACKEND___{{.*}}_
-Segmentation fault
+urMemGetInfoTest.Success/AMD_HIP_BACKEND___{{.*}}
+urMemGetInfoTest.InvalidNullPointerParamValue/AMD_HIP_BACKEND___{{.*}}
```

Comment:
These are returning early in the adapter, see here https://github.com/oneapi-src/unified-runtime/blob/adapters/source/loader/layers/validation/ur_valddi.cpp#L1350 so the adapter entry point is never called